bool record
---
float64[12] state
bool[4] peds_in_env